### ISSUE TITLE
How to run it on a private dataset?

### ISSUE THREAD

**Issue by Yasintrkmn** [csiro-robotics/Pair-VPR#4]

**Comment by Catchip** [comment]
One approach is to adjust your training set to the GSV format and your validation set to the MSLS format, but some minor adjustments are still required.

**Comment by Yasintrkmn** [comment]
@Catchip Can you share some instructions about 'minor adjustments' to achieve it step by step?

**Comment by StephenHausler** [comment]
Hi @Yasintrkmn, are you wanting to just run inference on a custom dataset, or training?
If inference, you can probably use https://github.com/csiro-robotics/Pair-VPR/blob/main/pairvpr/datasets/inference/test_dataset.py
First you'll need to setup your custom dataset by following: https://github.com/gmberton/VPR-datasets-downloader (to rename the images such that their GPS locations are embedded in the file name).
